code/datums/components/crafting/ranged_weapon.dm

```diff
@@ -240,6 +240,48 @@
 	time = 30 SECONDS //contemplate for a bit
 	category = CAT_WEAPON_RANGED
 
+/datum/crafting_recipe/deagle_prime //When you factor in the makarov (7 tc), the toolbox (1 tc), and the emag (3 tc), this comes to a total of 17 TC or thereabouts. Igorning the 20k pricetag, obviously.
```

Comment:
Might be worth mentioning this addition specifically in the changelog/PR body as it seems like it was added since that was written.

Reply:
nyoops I forgot.